plugin.rb

```diff
@@ -25,6 +25,11 @@ def self.access_restricted(guardian, topic, user)
       ## set a bool for if the topic is locked to user
       hasBeenLocked = true
 
+      # if topic is not restricted, dont lock it
+      if !topic.custom_fields["topic_restricted_access"]
+        hasBeenLocked = false
+      end
+
```

Comment:
I did not realize this logic had been removed with my change. added it back. This is now ready for review